README.md

```diff
@@ -34,7 +34,12 @@ That's it. Simply add a reference to `SimpleImageIO/SimpleImageIO.csproj` to you
 
 ## Building on other platforms
 
-In theory, the package works on any platform. However, the native dependencies have to be built for each. Currently, the workflow has been set up and tested for 64 bit Windows and Linux (Ubuntu 20.04). Other platforms (like OSX) need to be built from source. For these, the [SimpleImageIO.csproj](SimpleImageIO/SimpleImageIO.csproj) file needs to be adjusted, instructions can be found in the comments of that file. The process should be a simple copy&paste operation, provided nothing goes south when building the C++ library.
+In theory, the package works on any platform.
+However, the native dependencies have to be built for each.
+Currently, the workflow has been set up and tested for Intel 64bit Windows, Linux (Ubuntu 20.04) and macOS 10.15.
```

Comment:
Best rephrase to x86-64bit as it also runs on AMD ;)

Reply:
True, x86-64 is probably more commonly known (even though Intel 64 and AMD64 are often used as more readable synonyms for x86-64).

Reply:
Fixed in 33b176e.